tests/conftest.py

```diff
@@ -351,7 +351,7 @@ def _python_ngraph_required(request):
 @pytest.fixture(scope='session')
 def _python_vpu_plugin_required(request):
     image = request.config.getoption('--image')
-    if request.config.getoption('--image_os') != 'winserver2019':
+    if 'win' in request.config.getoption('--image_os'):
```

Comment:
May be `not in` should be here?
```suggestion
    if 'win' not in request.config.getoption('--image_os'):
```

Reply:
thank you! my fault!